automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -176,6 +176,8 @@ class NotebookClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTest
           withWebDriver { implicit driver =>
             withNewNotebook(cluster, Python3) { notebookPage =>
               notebookPage.executeCell("""print("Hello Notebook!")""") shouldBe Some("Hello Notebook!")
+              notebookPage.saveAndCheckpoint()
+              notebookPage.shutdownKernel()
```

Comment:
For my edification, what flakiness symptoms were you seeing and how did they lead you to make this change?

@Qi77Qi

Reply:
see https://broadinstitute.slack.com/archives/C32UN4KK4/p1574755297211600...the error message says unexpected alert...But when I run the test locally, I notice one chrome tab was never closed properly, and I tried closing a tab on a running notebook, and see a pop, which led me to the solution.